CMakeLists.txt

```diff
@@ -440,9 +440,11 @@ option(WITH_CEPHFS "CephFS is enabled" ON)
 #option for Mgr
 option(WITH_MGR "ceph-mgr is enabled" ON)
 if(WITH_MGR)
-  set(Python_ADDITIONAL_VERSIONS 2.7)
-  find_package(PythonInterp 2.7 REQUIRED)
-  find_package(PythonLibs 2.7 REQUIRED)
+  set(MGR_PYTHON_VERSION "2.7" CACHE STRING "python runtime for running mgr plugins")
```

Comment:
could we put a comment above this line:

```
# MGR_PYTHON_VERSION specifies the required Python major version and minimum minor version
# within that major version.
# To build in a Python 3-only environment, override with, e.g. -DMGR_PYTHON_VERSION=3.6
```

Reply:
@smithfarm this setting set the *minimal* required version of python, not the major and minor version. say, if we need to use python3 for running mgr plugins, but do not care which minor version is:
```
-DMGR_PYTHON_VERSION=3
```

would suffice. i updated the wording of the doc string of this cached string. HTH.